src/act.ts

```diff
@@ -20,10 +18,8 @@ function getIsReactActEnvironment() {
   return globalThis.IS_REACT_ACT_ENVIRONMENT;
 }
 
-type Act = typeof reactTestRendererAct;
-
-function withGlobalActEnvironment(actImplementation: Act) {
-  return (callback: Parameters<Act>[0]) => {
+function withGlobalActEnvironment(actImplementation: ReactAct) {
```

Comment:
Is it even useful to pass it `actImplementation` as a param?

Reply:
hmmmm, good question, we've got our code from RTL and just sprinkled types and eslint ignores over it. 
I guess it's easier to keep it in sync with their implmentation.